.gitignore

```diff
@@ -49,3 +49,5 @@ test/reports/
 # Ignore NPM
 node_modules
 .byebug_history
+
+Gemfile.lock
```

Comment:
Umm, I don't think we need to ignore `Gemfile.lock` here, that'd be counterintuitive to its purpose. Rather, we'd want to avoid committing any changes to it.

For more information, check out: http://bundler.io/v1.3/rationale.html#sharing-your-application-with-other-developers

> When your co-developers (or you on another machine) check out your code, it will come with the exact versions of all the third-party code your application used on the machine that you last developed on (in the Gemfile.lock). When **they** run bundle install, bundler will find the Gemfile.lock and skip the dependency resolution step. Instead, it will install all of the same gems that you used on the original machine.